lib/pages/login_page.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/pages/widgets/blur_background.dart';
+import 'package:flutter_survey/pages/widgets/dimmed_background.dart';
+import 'package:flutter_survey/resouces/assets.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class LoginPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      resizeToAvoidBottomInset: false,
+      child: Stack(
+        children: [
+          Container(
+            decoration: BoxDecoration(
+              image: DecorationImage(
+                image: AssetImage(Assets.bgLogin),
+                fit: BoxFit.cover,
+              ),
+            ),
+          ),
+          BlurBackground(),
+          DimmedBackground(
+            colors: [
+              Colors.black.withOpacity(0.2),
+              Colors.black.withOpacity(0.8),
+              Colors.black,
+            ],
+            stops: [0.0, 0.6, 1.0],
```

Comment:
maybe consider adding these as the default values of the widget? 🤔  just want to make it shorter when using in here

Reply:
I would like to keep `DimmedBackground` or `BlurBackground` as a generic implementation, let me move this into a const instead